test/framework-ondata-mocha.js

```diff
@@ -430,8 +430,8 @@ describe("framework-ondata", function() {
 
             // Define the fRunUpdater that framework should invoke
             var fRunUpdater = function( vni ) { 
-               // Should not call fRunUpdater when only optional data has been sent
-               // and required port data is still missing
+               // Should not call fRunUpdater when data is missing 
+               // from an attached port
 console.log('in fRunUpdater');
```

Comment:
Should probably remove the console line.  We do not need that since the fail will shut it down if the updater executes.